### PR TITLE
[Fixes #15106] Document how to properly handle modals with dynamic height

### LINKED ISSUE #15106: Document how to properly handle modals with dynamic content height
Modals with dynamic content height should call `$('#myModal').data('bs.modal').handleUpdate()` whenever the content updates. Only necessary when the update happens while it is shown though.

### PR DESCRIPTION
**Pull request by kkirsche**
Fixes #15106 — Document how to properly handle modals with dynamic content height

X-Ref: #15881 — Fix backdrop not readjusting when height changes

### CODE REVIEW

**Comment by mdo** [comment]
Works for me.

**Comment by hnrch02** [comment]
See my comment here for clarification: https://github.com/twbs/bootstrap/pull/15881#issuecomment-75883495

**Comment by hnrch02** [comment]
But documenting it is a good idea, just need to change the text to reflect the new behavior.

File: docs/_includes/js/modal.html [hunk 1]
```diff
@@ -394,6 +394,9 @@ <h4 class="modal-title" id="exampleModalLabel">New message</h4>
 })
 {% endhighlight %}
 
+  <h2 id="modals-handle-update">Modals with dynamic heights</h2>
+  <p>Modals which have a dynamic height such as modals loading their content with AJAX should call <code>$('#myModal').data('bs.modal').handleUpdate()</code> when the content updates to ensure the modal and backdrop are the correct sizes. This only needs to occur if the modal is visible when the update to the modal's content occurs.</p>
```
Comment (hnrch02) [inline docs/_includes/js/modal.html]: The second sentence repeats "occur", BTW.

**Comment by kkirsche** [comment]
Updated verbiage based on @hnrch02's comments.

**Pull request closed without merging**